app/controllers/admin/unattached_promo_registrations_controller.rb

```diff
@@ -85,6 +88,10 @@ def assign_installer_type
 
   private
 
+  def sortable_columns
```

Comment:
Is this method being used?

Reply:
Yeah it's used in https://github.com/brave-intl/publishers/blob/staging/app/helpers/admin_helper.rb#L4 which calls https://github.com/brave-intl/publishers/blob/staging/app/controllers/admin_controller.rb#L24